//try Catch 

const name =  'ami';
try{
    name = 'tmi';
}
catch(error){
    console.log(error);
}
console.log(name);
